test/functional/cmdLineTests/pltest/playlist.xml

```diff
@@ -46,6 +45,30 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<subset>11</subset>
 		</subsets>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_pltest_osx</testCaseName>
```

Comment:
- The command for `cmdLineTester_pltest` and `cmdLineTester_pltest_osx` are the same. Why we create a new test `cmdLineTester_pltest_osx` for osx and disable it? 
Original `cmdLineTester_pltest` is excluded on osx. If #3685 is fixed and the new issue is tracked in #4520, could we just update the comment in `cmdLineTester_pltest`?

- Question related to #4520, giving the fact that `pltest` worked on other platforms and failed on osx. Does pltest send `SIGQUIT` when running on other platforms?

Reply:
fyi - @llxia 
- `cmdLineTester_pltest_osx` is different. It excludes `j9sock` tests on OSX: `-exclude:j9sock`. it is disabled on OSX due to #4520.
- Yes, `SIGQUIT` is raised on other platforms in `j9process_testNewProcessGroupByTriggeringSignal`, which is a testcase in the `pltest` suite. Refer to https://github.com/eclipse/openj9/issues/4520#issuecomment-459092010 for the hypothesis on why the test framework crashes on OSX due to `SIGQUIT` from `pltest`.
https://github.com/eclipse/openj9/blob/ab007729a4388b5ff22b36c8c4e16c21b30c27f1/runtime/tests/port/j9processTest.c#L488-L492

Reply:
added a comment for merging `cmdLineTester_pltest` and `cmdLineTester_pltest_osx`.